### PR TITLE
[sample_multi_transcode] Enable tiles for HEVC encoder

### PR DESCRIPTION
**Pull request by zoss9999**
This commit is to enable "tiles" for HEVC encoder in sample_multi_transcode.

### CODE REVIEW

File: samples/sample_multi_transcode/include/pipeline_transcode.h [hunk 1]
```diff
@@ -797,6 +797,7 @@ namespace TranscodingSample
 
         // HEVC
         mfxExtHEVCParam          m_ExtHEVCParam;
+        mfxExtHEVCTiles          m_ExtHEVCTiles;
```
Comment (mgonchar) [inline samples/sample_multi_transcode/include/pipeline_transcode.h]: You need to add initialization of new member to constructor

Comment (zoss9999) [reply samples/sample_multi_transcode/include/pipeline_transcode.h]: Thanks, I fixed it.

Comment (mgonchar) [reply samples/sample_multi_transcode/include/pipeline_transcode.h]: I suggest to merge your two commits to one. Now there is a state in git history when this feature is in some intermediate state

**Comment by zoss9999** [comment]
broken fork. will reopen pull request

**Pull request closed without merging**